src/transformers/configuration_reformer.py

```diff
@@ -115,6 +115,8 @@ class ReformerConfig(PretrainedConfig):
         vocab_size (:obj:`int`, optional, defaults to 320):
             Vocabulary size of the Reformer model. Defines the different tokens that
             can be represented by the `inputs_ids` passed to the forward method of :class:`~transformers.ReformerModel`.
+        tie_word_embeddings (:obj:`bool`, optional, defaults to False):
```

Comment:
```suggestion
        tie_word_embeddings (:obj:`bool`, `optional`, defaults to :obj:`False`):
```
I know this is not consistent with the rest of this file, but I haven't gotten to updating the model/config docstrings yet. Let's always use our preferred format (optional in italics, all object in :obj: backquotes) when making new contributions.

Reply:
Of course! Can you please explain why? If we are turning "optional" into a code string, shouldn't it be `Optional` then to match  `typing.Optional` or is this for a different reason? i.e. just trying to understand the convention and it's much easier to remember to do when I know why this is so ;) Thank you.

Reply:
No restructured text is not Markdown. `optional` will be rendered in italics (you need double backquotes for code-formatting)

Reply:
oh, I had no idea - I'm totally new to rst - thank you!